test_databaseconnect.py

```diff
@@ -95,44 +37,13 @@ def test_get_question_response_with_subject(self):
         assert type(response) is tuple
 
     def test_add_learnt_statement_to_database(self):
-        db = databaseconnect.add_learnt_statement_to_database("subject", "root", "verb")
-        cursor = db.cursor()
-        cursor.execute(
-            "select * from question_table where subject='subject' and root_word='root' and verb='verb'"
+        res = databaseconnect.add_learnt_statement_to_database(
+            "subject", "root", "verb"
         )
-        res = cursor.fetchone()[0]
-        assert res == 1
+        # test not needed
+        # handled by test_add_to_database_of_statement_table()
+        assert res
 
     def test_learn_question_response(self):
         response = databaseconnect.learn_question_response("H")
         assert type(response) is tuple
-
-    def test_clear_table_with_chat_table(self, monkeypatch):
-        from io import StringIO
-
-        yes = StringIO("y\n")
-        monkeypatch.setattr("sys.stdin", yes)
-
-        db = databaseconnect.clear_table("chat_table")
-
-        cursor = db.cursor()
-        cursor.execute("select * from chat_table")
-        entries = cursor.fetchone()
-        assert entries is None
-
-    def test_clear_table_with_statement_or_question_table(self, monkeypatch):
-        from io import StringIO
-
-        yes = StringIO("y\n")
-        monkeypatch.setattr("sys.stdin", yes)
-
-        db = databaseconnect.clear_table("statement_table")
-
-        cursor = db.cursor()
-        cursor.execute("select * from statement_table")
-        entries_1 = cursor.fetchone()
-
-        cursor.execute("select * from question_table")
-        entries_2 = cursor.fetchone()
-
-        assert entries_1 is None and entries_2 is None
```

Comment:
Functions restricted to developer use; not used by bot itself. Thus, tests not needed.